### PR TITLE
add chainsaw service description

### PR DESCRIPTION
**Pull request by iuwqyir**
## Problem solved

Added chainsaw service description so it can be used for auth

## Contributor NFT

Paste in your wallet address below and we will airdrop you a special NFT when your pull request is merged. 

```Address: ```

---

## PR-Codex overview
This PR adds a new service called `chainsaw` to the `service-utils` package, updates dependencies, and removes unnecessary code.

### Detailed summary
- Added `chainsaw` service definition in `service-utils`
- Updated `@thirdweb-dev/service-utils` version to `^0.4.36`
- Removed unnecessary code related to `HIDDEN_SERVICES`
- Updated dependencies in `pnpm-lock.yaml` for `@thirdweb-dev/service-utils@0.4.36`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/service-utils/src/core/services.ts [hunk 1]
```diff
@@ -51,6 +51,13 @@ export const SERVICE_DEFINITIONS = {
     // all actions allowed
     actions: [],
   },
+  chainsaw: {
+    name: "chainsaw",
+    title: "Chainsaw",
```
Comment (jnsdls) [inline packages/service-utils/src/core/services.ts:56]: probably not the public name?

**Review by jnsdls — COMMENTED** [review]
tried it and it fails on api server requirements so blocked on api server for now

File: packages/service-utils/src/core/services.ts (hunk 1 above)
Comment (iuwqyir) [reply packages/service-utils/src/core/services.ts:56]: Probably not, but can be changed after. Don't want to wait on naming.

Figured out api-server needs changes too so just created a PR in api-server

Comment (jnsdls) [reply packages/service-utils/src/core/services.ts:56]: yep, doesn't show in UI for now, right?

Comment (jnsdls) [reply packages/service-utils/src/core/services.ts:56]: and api server PR makes sense, however I think you'll have to add it to `SERVICES` in service utils (in this repo), too?

**Review by joaquim-verges — APPROVED** [review]

File: packages/service-utils/src/core/services.ts (hunk 1 above)
Comment (iuwqyir) [reply packages/service-utils/src/core/services.ts:56]: `SERVICES` are generated from `SERVICE_DEFINITIONS`, if it's the same file you mean

https://github.com/thirdweb-dev/js/blob/main/packages/service-utils/src/core/services.ts#L60

Comment (iuwqyir) [reply packages/service-utils/src/core/services.ts:56]: Added chainsaw to HIDDEN_SERVICES [857b040](https://github.com/thirdweb-dev/js/pull/3745/commits/857b040b76e56589b5ac8425f83b500c82eb361d)

**Review by jnsdls — APPROVED** [review]

**Pull request merged**